bindata/external-secrets/external-secrets-namespace.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    openshift.io/cluster-monitoring: "true"
```

Comment:
I don't know if you want to really use cluster-monitoring,
I saw that https://github.com/openshift/cert-manager-operator/blob/master/bindata/cert-manager-deployment/cert-manager-namespace.yaml has it too, but here's the discussion thread which has some opinionated views to review before considering:
- https://redhat-internal.slack.com/archives/C04JVEQ8C79/p1721033592437669
- https://redhat-internal.slack.com/archives/CH76YSYSC/p1703660194599209

Reply:
Yeah, better to add it to docs, alongside operator section?